### PR TITLE
Start Olathe KS file

### PR DESCRIPTION
**Pull request by githubteacher**
### Description of Change

- Creating a file for Olathe KS

### Test results

- I did not do any testing

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: olathe-ks.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Olathe, KS
```
Comment (githubteacher) [inline olathe-ks.md:1]: Olathe is great, I hear it's over 40 there today! I'm glad I'm in Luxembourg.

**Review by githubteacher — COMMENTED** [review]
👍

**Comment by FloSeguin** [comment]
great

**Review by mj580 — APPROVED** [review]
approved!

**Pull request merged**